hrpc/call.go

```diff
@@ -205,6 +205,49 @@ func MaxVersions(versions uint32) func(Call) error {
 	}
 }
 
+// MaxResultsPerColumnFamily sets the maximum number of values returned for each row
+func MaxResultsPerColumnFamily(maxresults uint32) func(Call) error {
+	opt := "'MaxResultsperColumnFamily' "
+	return func(g Call) error {
+		switch c := g.(type) {
+		default:
+			return errors.New(opt + "option can only be used with Get or Scan queries")
+		case *Get:
+			if maxresults > math.MaxInt32 {
+				return errors.New(opt + "exceeds supported number of value results")
+			}
+			c.storeLimit = maxresults
+		case *Scan:
+			if maxresults > math.MaxInt32 {
+				return errors.New(opt + "exceeds supported number of value results")
+			}
+			c.storeLimit = maxresults
+		}
+		return nil
+	}
+}
+
+// ResultOffset sets the offset within a column family set of results
```

Comment:
how about "ResultOffset is a option for Scan or Get requests that sets the offset for cells within a column family"